packages/@vuepress/theme-default/src/client/styles/layout.scss

```diff
@@ -160,7 +161,6 @@ h6 {
     top: 0;
     padding-top: var(--navbar-height);
     transform: translateX(-100%);
-    transition: transform 0.2s ease;
```

Comment:
Removed here and moved to [L37](https://github.com/vuepress/vuepress-next/pull/321/files#diff-4dd3e0f55ac29286ecca1b5abe7786ded54987ea3057f5bcd7cf4498823b8734R37)

This `transition` in `@media` will override the normal `transition` for `background-color` and `border-right-color`